frontend/src/app/components/amount/amount.component.ts

```diff
@@ -19,7 +19,7 @@ export class AmountComponent implements OnInit, OnDestroy {
   currencySubscription: Subscription;
 
   @Input() satoshis: number;
-  @Input() digitsInfo = '1.8-8';
+  @Input() digitsInfo = '1.0-8';
```

Comment:
I think we need to really think through this change.

I don't like how some Bitcoin services remove the zeroes make it difficult to see how many satoshis it actually was.. for example

`0.0000001`

Is this 1 satoshi or 10 satoshis?

Reply:
yes must always show 8 digits

Reply:
I think it's not even necessary anymore. Was from a previous commit. Will remove.